pkg/status/status.go

```diff
@@ -0,0 +1,5 @@
+package status
+
+var Secured bool
+var SessionName string
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported var SessionName should have comment or be unexported](https://app.codacy.com/app/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4570750)